backend/routes/creators.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const jwt = require('jwt-simple');
+
+const{Creator}=require('../data/models');
+const router = express.Router();
+router.use(express.json());
+
+router.get('/:id', (req, res, next)=>{
+    Creator.findOne({where: {id: req.params.id} })
+    .then((creator) => res.send(creator))
+    .catch(next);
+});
+
+router.delete('/:id', (req, res, next)=>{
+    const { id } = req.params;
+    console.log({ deleting: id })
+    Creator.findByPk(id)
+        .then((creator) => creator.destroy())
+        .then(() => res.sendStatus(204))
+        .catch(next);
+});
+
+router.post('/', (req, res, next) => {
```

Comment:
do we have another use case (aside from 1st login) when we're adding a Creator? Right now, adding a creator is being handled through https://github.com/StellarApp/capstone-influencer-search/blob/9ca48a52b1ad2cb7ca81e6104d1c3878304e5ccc/backend/routes/auth/facebook.js#L4

Reply:
We don't need this one

Reply:
yup... but i imagine we'll do it using `put` or `patch`. Also the code inside this route uses `Creator.create`.